full-service/src/config.rs

```diff
@@ -213,6 +213,7 @@ impl PeersConfig {
             .iter()
             .map(|client_uri| {
                 ThickClient::new(
+                    "".to_string(),
```

Comment:
We should add a new command line argument `--chain-id` and pass it here.

Reply:
ah okay, yeah that works

Reply:
Optional? Default value?

Reply:
Nope, required

Reply:
ah okay, thanks! Will add that as a required cla

Reply:
Where can I find info on chain-ids?

Reply:
https://github.com/mobilecoinfoundation/mcips/blob/main/text/0049-chain-ids.md